src/Features/CSharp/Portable/UseSimpleUsingStatement/UseSimpleUsingStatementCodeFixProvider.cs

```diff
@@ -131,12 +133,12 @@ private static SyntaxTriviaList Expand(List<StatementSyntax> result, UsingStatem
 
         private static LocalDeclarationStatementSyntax Convert(UsingStatementSyntax usingStatement)
         {
-            return SyntaxFactory.LocalDeclarationStatement(
+            return LocalDeclarationStatement(
                 usingStatement.AwaitKeyword,
                 usingStatement.UsingKeyword,
                 modifiers: default,
                 usingStatement.Declaration,
-                SyntaxFactory.Token(SyntaxKind.SemicolonToken));
+                Token(SyntaxKind.SemicolonToken)).WithTrailingTrivia(ElasticCarriageReturnLineFeed);
```

Comment:
the core issue here was that we were making a LocalDecl with jsut an elastic trailing trivia.  That ended up causing the new code to look like:

// starting code:
```c#
using (var v = x)
{
    FirstStatement(
        a, b, c);
    SecondStatement();
}
```

// converted to:
```c#
using var v = x; FirstStatement(
        a, b, c);
    SecondStatement();
```

So when teh formatter fixed ths up, it tried to move `FirstStatement` back to align with the `using`.  So that pushed back all the ars to the left as well.

Trivial fix here is to put on an elastic newline so that we instead have:

```c#
using var v = x;
    FirstStatement(
        a, b, c);
    SecondStatement();
```

This then easily formats to:

```c#
using var v = x;
FirstStatement(
    a, b, c);
SecondStatement();
```